posts/2023-09-29-meetup-at-the-rotterdam-office-|-devoxx-try-out:-maintaining-maven-at-scale-&-machine-learning-in-java.adoc

```diff
@@ -0,0 +1,129 @@
+= Meetup At The Rotterdam Office | Devoxx Try-out: Maintaining Maven at scale & Machine Learning in Java
+PeuTit
+v1.0, 2023-09-29
+:title: Meetup At The Rotterdam Office | Devoxx Try-out: Maintaining Maven at scale & Machine Learning in Java
+:lang: en
+:tags: [java, ml, oss, devoxx, en]
+
+Did you miss your chance to snag Devoxx tickets this time around? Are you eager to get a sneak peek into the Devoxx experience? We've got just the solution for you!
+
+Welcome to the **Devoxx Try-out**, this meetup was an exclusive pre-event proudly brought to you by us.
+
+You can find the complete schedule of the meet-up here:
+
+🕔 **17:00**: Doors Open, Food, Drinks, and a Warm Lunatech Welcome!
+
+🗣️ **18:30**: **Maintaining Maven at Scale (by Tim te Beek)**
+
+🗣️ **19:30**: **Lessons Learned using Machine Learning in Java (by Jago de Vreede)**
+
+🍹 **20:30**: Networking Time! Grab a Drink and Connect.
+
+🕤 **21:30**: It's Time to Bid Farewell.
```

Comment:
Maybe no need to include the complete schedule. You can just give the title of the two talks and the names of the speakers